swmmio/core.py

```diff
@@ -857,6 +876,31 @@ def xsections(self, df):
         """Set inp.xsections DataFrame."""
         self._xsections_df = df
 
+
+    @property
+    def lid_usage(self):
+        """
+        Get/set LID_USAGE section of the INP file.
+        """
+        if self._lid_usage_df is None:
+            self._lid_usage_df = dataframe_from_inp(self.path, "[LID_USAGE]")
+        return self._lid_usage_df
+
+    @property
+    def raingages(self):
```

Comment:
This raingages getter section is already defined above around line 731. Please remove this property.